docker/api-gateway/conf.d/default.conf

```diff
@@ -7,4 +7,10 @@ server {
         proxy_set_header Host            $host;
         proxy_set_header X-Forwarded-For $remote_addr;
     }
+
+    location /api/job {
```

Comment:
We already renamed `/file` to `/files` in the parallel commit, let's do the same for this service as well: `/api/job` => `/api/jobs`